### DISCUSSION TITLE
Can watchtower notify by email but not update?

### DISCUSSION THREAD

**Discussion by TheCableGuy99** [containrrr/watchtower#1580] [Q&A] [answered]
Hi guys,

I was wondering if Watchtower can simply notify me by SMTP if updates are available so I can manage the updates myself?

The reason is that now and again I find things are down because something has gone wrong at some point.

Thanks.

**Comment by TheCableGuy99** [comment]
nevermind, I resolved this.

**Comment by simonaronsson** [comment, marked as answer]
For any others who wonder about this, the answer is no. While watchtower is able to pull the image but refrain from recreating the container, it is not able to only check without pulling the new image.

**Reply by piksel** [reply]
With the work done for the HEAD requests, it should be possible to enable this feature without too much effort. Not something we are working on right now though.

**Comment by TheCableGuy99** [comment]
Unless i'm mistaken (quite possible) I'm fairly sure I did actually manage this at the time (I have disabled watchtower globally for the time being).

I set SMTP notifiations which work but also turned set the "com.centurylinklabs.watchtower.enable" flag to flase on the ones I didn't want it to update.

As said I could be wrong as I need it disabled for the time being so cannot double check but this way I could have swore it notified me but not actually perform the update.... could you double confirm please?
